src/directive/model.ts

```diff
@@ -35,6 +35,7 @@ export interface Directive {
   updatedDt: Date;
   trigger: DirectiveTrigger;
   stewardAccountId?: string;
```

Comment:
Commented on the commit already (unintentionally, I didn't know you could do that until just now), but I think we should remove the `stewardAccountId` if it's only used in tests; tests shouldn't be able to influence what our models look like.

Reply:
Done.